continuous-integration/linux/lint-cpp.sh

```diff
@@ -11,8 +11,8 @@ fi
 
 #Todo: buildDir=$1
 
-cppFiles=$(find "$ROOT_DIR" -name '*.cpp' |grep --invert-match src/3rd-party |grep --invert-match _skbuild)
-hFiles=$(find "$ROOT_DIR" -name '*.h' |grep --invert-match src/3rd-party |grep --invert-match _skbuild)
+cppFiles=$(find "$ROOT_DIR" -name '*.cpp' | grep --invert-match src/3rd-party | grep --invert-match _skbuild | grep --invert-match build)
```

Comment:
I'm assuming this change was made so that you can have a local build folder to get intellisense in IDE?

Reply:
yep :+1: